ckan/controllers/user.py

```diff
@@ -622,6 +615,13 @@ def logged_in(self):
             else:
                 return self.login(error=err)
 
+    def me(self, locale=None):
+        if not c.user:
+            h.redirect_to(locale=locale, controller='user', action='login',
+                          id=None)
+        user_ref = c.userobj.get_reference_preferred_for_uri()
+        h.redirect_to('/')
+
```

Comment:
why has this moved so far down in the file?

Reply:
moved code back